### ISSUE TITLE
MIT License from github.com/spdx not detected

### ISSUE THREAD

**Issue by mbarkhau** [licensee/licensee#395]
### Describe the bug

I obtained the text for the MIT license from this file: https://github.com/spdx/license-list-data/blob/master/text/MIT.txt

This text differs in the second paragraph from the one used by licensee.

```diff
 - The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
 + The above copyright notice and this permission notice (including the next paragraph) shall be included in all copies or substantial portions of the Software.
```
I have traced this change back to the following commit: https://github.com/spdx/license-list-XML/commit/ca17b9160aab8acce72fe62720e63f68a782406b and the following issue: https://github.com/spdx/license-list-XML/issues/729

Tested using master
```
$ git show HEAD
commit cfc113451547b0fe0527c78793ae43536090bc24

$ bundle exec bin/licensee version
9.11.0

$ bundle exec bin/licensee detect ../markdown-katex/LICENSE
License:        NOASSERTION
Matched files:  LICENSE
LICENSE:
  Content hash:  ecbca5288cf69bc79f58edfb26f0f6e0fdac3c7c
  License:       NOASSERTION
```

**Comment by mlinksva** [comment]
Yes it's because of the added optional text. I haven't gotten around to filing an issue yet, but I'd prefer SPDX distinguish between preferred optional and just optional text, and only render the former in non-annotated formats, such as plain text. This addition is definitely in the latter category, it's pretty rare and superfluous AFAIK.

If there's some evidence of it being non-rare, I suppose licensee's content normalization step could strip it out.

**Comment by mbarkhau** [comment]
In other words, there is no plan to detect licenses with this paragraph as MIT?

**Comment by mlinksva** [comment]
@mbarkhau see my last sentence above. However, looking at https://github.com/spdx/license-list-XML/issues/729 again, I guess there's evidence there, and the fact that it has been published by SPDX will generate more. I'll make a PR momentarily.

**Comment by mlinksva** [comment]
But note that doesn't mean there's a plan for github.com to detect this variation. #396 (or a different implementation) would need to be merged, a new release of licensee cut, and that new release deployed on github.com. So if you really just want the license of your repository to be detected ASAP, your best bet is to switch to using MIT without the optional text.

**Comment by mbarkhau** [comment]
> your best bet is to switch to using MIT without the optional text
Yes of course, thanks for the update :+1:
